### ISSUE TITLE
CC should disappear once focus is out of the graph.

### ISSUE THREAD

**Issue by situlindia** [julianna-langston/chart2music#687]
The CC text still remains and doesn't go off even though focus has been changed.

**Comment by julianna-langston** [comment]
We don't clear the CC when the chart loses focus, in case the screen reader user wants to further interrogate the text. That said, we could update an attribute on the CC element to indicate whether or not the chart has focus. Then, at least developers could hide the CC when the element is out of focus (or always, if they want).

@spmealin , Thoughts?

**Comment by spmealin** [comment]
I don't think that the CC should be hidden upon focus loss by default. It is sometimes handy to be able to copy/read the CC with screen reader commands. I do like having the ability for the chart author to optionally enable "clear CC when focus is lost" functionality.
Could this be supported by passing in an option when the chart is created? That way, we give authors the ability to clear CC on focus loss, but in the future, we could add an option in the options dialog so the end-user can overwrite this behavior.

**Comment by julianna-langston** [comment]
Just to close the loop - If anyone is interested in how to visually clear the CC text on blur, we've documented the CSS pattern using `:focus-wthin`. This preserves the CC in the accessibility tree, so screen-reader users can still review the generated text after leaving the chart, while allowing sighted users to see it only during chart interaction. 
See [CC styling: Visible only when the chart is in focus](https://www.chart2music.com/docs/API/CCStyling/#visible-only-when-the-chart-is-in-focus). The same page also covers [keeping a CC always visually hidden](https://www.chart2music.com/docs/API/CCStyling/#always-invisible) without using display: none.
